scripts/buildVM-files/runQemuVM.sh

```diff
@@ -62,4 +94,6 @@ qemu-system-x86_64 \
 	-drive if=pflash,format=raw,readonly,file="$SCRIPT_DIR/OVMF/OVMF_CODE.fd" \
 	-drive if=pflash,format=raw,file="$SCRIPT_DIR/OVMF/OVMF_VARS.fd" \
 	-drive file="$SCRIPT_DIR/${VM_NAME}.qcow2",index=0,media=disk \
+	-device e1000,netdev=nilrt_net0,mac=$macaddr \
```

Comment:
Should this e1000 driver name also be configurable from the command line?

Reply:
I think the 99% use-case for this shell script in particular, is by an interactive user who is trying to start a VM quickly. There might be some cases where we would want to emulate a particular PHY driver for ATS purposes, but I don't see an argument in this script being used very frequently.